migration-roadmap/search.md

```diff
@@ -154,125 +201,234 @@
++            focusable="false"
++            aria-hidden="true"
++            role="img"
++        >
++            <title>Search</title>
++            <use
++                xlink:href="#spectrum-icon-18-Search"
++                href="#spectrum-icon-18-Search"
++            />
++        </svg>
++    </button>
++</form>
+```
 
-- Collapsed/expanded state support - web component should support animated expansion/collapse
-- More sophisticated state management for search interactions
+</details>
 
-**Web Component Features Missing from CSS:**
+### CSS => SWC mapping
 
-- `action` attribute - CSS should provide styling for form action states
-- `method` attribute - CSS should provide styling based on form method
-- `holdValueOnEscape` attribute - CSS should provide styling for this behavior state
-- Help text slots - CSS should provide styling for slotted help text elements
+| CSS selector                                                                          | Attribute or slot          | Status                       |
+| ------------------------------------------------------------------------------------- | -------------------------- | ---------------------------- |
+| `.spectrum-Search--sizeS`, `.spectrum-Search--sizeL`, `.spectrum-Search--sizeXL`      | `size`                     | Implemented                  |
+| `.spectrum-Search.is-disabled`                                                        | `disabled`                 | Implemented                  |
+| `.spectrum-Search .spectrum-HelpText`                                                 | `help-text` slot           | Implemented                  |
+| `.spectrum-Search:lang(ja)`, `.spectrum-Search:lang(ko)`, `.spectrum-Search:lang(zh)` | Language-specific styling  | Implemented                  |
+| `.spectrum-Search .spectrum-Search-clearButton`                                       | Clear button functionality | Implemented                  |
+| `.spectrum-Search .spectrum-Search-textfield`                                         | Textfield functionality    | Implemented                  |
+| `.spectrum-Search .spectrum-Search-icon`                                              | Search icon                | Implemented                  |
+| `.spectrum-Search .spectrum-Search-input`                                             | Input element              | Implemented                  |
+|                                                                                       | `action`                   | Missing from CSS             |
+|                                                                                       | `method`                   | Missing from CSS             |
+|                                                                                       | `holdValueOnEscape`        | Missing from CSS             |
+|                                                                                       | `negative-help-text` slot  | Missing from CSS             |
```

Comment:
"Missing from CSS" doesn't feel super valuable for our purposes, taking this out if no one is opposed.